app/components/candidate_interface/offer_review_component.rb

```diff
@@ -60,5 +76,17 @@ def course_row_value
           tag.p(@course_choice.current_course.description, class: 'govuk-body')
       end
     end
+
+    def fee_row_value
+      course = @course_choice.current_course
+
+      if course.fee_international && course.fee_domestic
+        tag.p("UK Students: £#{course.fee_domestic}") + tag.br("International Students: £#{course.fee_international}") + tag.br(course.fee_details, class: 'govuk-body')
```

Comment:
I think the brackets are slightly wrong here, and the `<br>` element shouldn't contain any content (as it just represents a line break.

The "fee details" field can contain multiple paragraphs (it's rendered [using markdown on Find](https://github.com/DFE-Digital/find-teacher-training/blob/main/app/components/courses/fees_component.html.erb#L35)) so we might need to do the same here I think... (it's also optional so not always present).

```suggestion
        tag.p("UK Students: £#{course.fee_domestic}" + tag.br + "International Students: £#{course.fee_international}", class: 'govuk-body') + tag.br + markdown(course.fee_details)
```